pages/static/tools/manifest.webmanifest

```diff
@@ -0,0 +1,6 @@
+{
+  "icons": [
```

Comment:
I'm not sure, but I think the `name` is a required field for the manifest. Or am I wrong?

Reply:
Good point.
Not sure about general recommendations. But we are definitely going to add more manifest info in the future.
I think that it's ok for the current PR to have just icons to be defined.